### PR TITLE
BITMAKER-1261: Improve Readme

### PR DESCRIPTION
**Pull request by Adriana618**
#### Ticket url:
- http://tasks.bitmaker.la/issues/1261

#### Description:
- Improve Readme, also modified commands to get data all from job and filter jobs for tag. Now we can get data with
`bitmaker data job <jobID> <spiderID> <projectID>`
The job data is saved in a folder called projects_path in json format. This folder is created in the path relative to where the terminal is.
- We can get jobs filter by tag
`bitmaker list job <spiderID> -t <tag>`
Tag is optional.
#### Checklist:
- [X] Rebase my branch

### CODE REVIEW

**Comment by Adriana618** [comment]
@mgonnav This is ready for review.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,38 @@
-# Bitmaker Cloud CLI
+<h1 align="center"> Bitmaker Cloud CLI </h1>
 
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+Bitmaker Cloud CLI is the command line client to interact with Bitmaker Cloud API. Allows the user to perform the following actions:
+- Link a Scrapy project with a project in Bitmaker Cloud.
+- Create projects, jobs and cronjobs in Bitmaker Cloud.
```
Comment (mgonnav) [inline README.md:7]: Currently, we can't create cronjobs via the client 😅

**Review by mgonnav — COMMENTED** [review]
Besides the only comment I added, everything else (retrieving job data, listing jobs by tag and showing job tags when listing jobs) is working well.

File: README.md (hunk 1 above)
Comment (Adriana618) [reply README.md]: Sure... I removed

**Review by mgonnav — APPROVED** [review]

**Pull request merged**